adapter/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -64,6 +64,7 @@ SqlNode SqlCreateMaterializedView() :
      SqlIdentifier cubeName;
      boolean ifNotExists = false;
      SqlSelect query;
+     SqlNodeList orderList;
```

Comment:
Have lost the context I have over the grammar, so trying to understand the need for a SqlNodeList ? 
Is this grammar representation for all Materialized Views of a dataset together and not per view ? (and hence the list)

Reply:
`SqlSelect` doesn't allow you to specify an `ORDER BY`. It only allows a basical SELECT with `GROUP BY`.